avocado/core/nrunner/runnable.py

```diff
@@ -170,18 +170,15 @@ def config(self, config):
         :param config: A config dict with new values for Runnable.
         :type config: dict
         """
-        command = self.pick_runner_command(self.kind)
-        if command is not None:
-            command = " ".join(command)
-            configuration_used = STANDALONE_EXECUTABLE_CONFIG_USED.get(command)
-            if not set(configuration_used).issubset(set(config.keys())):
-                LOG.warning(
-                    "The runnable config should have only values "
-                    "essential for its runner. In the next version of "
-                    "avocado, this will raise a Value Error. Please "
-                    "use avocado.core.nrunner.runnable.Runnable.filter_runnable_config "
-                    "or avocado.core.nrunner.runnable.Runnable.from_avocado_config"
-                )
+        configuration_used = Runnable.get_configuration_used_by_kind(self.kind)
+        if not set(configuration_used).issubset(set(config.keys())):
+            LOG.warning(
```

Comment:
This is not really related to this PR, but IMO it's time to remove this warning and just raise a Value Error. What do you think?

Reply:
Yes, the idea is to do that after 99.0 (so the warning will be gone in 100.0).  Thanks!